Source/Core/Core/ActionReplay.cpp

```diff
@@ -604,7 +606,7 @@ static bool ZeroCode_MemoryCopy(const u32 val_last, const ARAddr& addr, const u3
     {  // Memory Copy With Pointers Support
       LogInfo("Memory Copy With Pointers Support");
       LogInfo("--------");
-      for (int i = 0; i < 138; ++i)
+      for (int i = 0; i < num_bytes; ++i)
```

Comment:
The hard-coded 138 value was copied from an example in the spec. Additionally, in the spec example, it was hexadecimal, not decimal. The actual length is obviously specified by the code.